fontforge/tottfgpos.c

```diff
@@ -1929,7 +1929,7 @@ static void dumpGSUBligdata(FILE *gsub,SplineFont *sf,
 	putshort(gsub,pcnt);
 	if ( pcnt>=max ) {
 	    max = pcnt+100;
-	    ligoffsets = realloc(ligoffsets,max*sizeof(int));
+	    ligoffsets = realloc(ligoffsets,max*sizeof(uint16));
```

Comment:
This one is "unsafe" as an individual line, but in context it looks correct. (See line 1906 and how it's used as an array.)

Reply:
Reverted.